pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -59,6 +59,13 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 	if !validateTargetThresholds(targetThresholds) {
 		return
 	}
+	// check if CPU/Mem not set in targetThresholds and set it to 100
```

Comment:
check if CPU/Mem not set in targetThresholds and set it to 100
check if CPU/Mem are set in targetThresholds, if not, set them to 100